tests/tasks/setup_mock_wifi_wpa3_sae.yml

```diff
@@ -0,0 +1,72 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: install hostapd and mac80211_hwsim kernel module in RHEL 8
+  shell: |
+    yum-config-manager --add-repo http://download.eng.brq.redhat.com/rhel-8/rel-eng/RHEL-8/latest-RHEL-8.3.0/compose/BaseOS/$(uname -p)/os
+    rpm --import http://download.eng.brq.redhat.com/rhel-8/rel-eng/RHEL-8/latest-RHEL-8.3.0/compose/BaseOS/$(uname -p)/os/RPM-GPG-KEY-redhat-release
+    yum-config-manager --add-repo http://download.eng.brq.redhat.com/rhel-8/rel-eng/RHEL-8/latest-RHEL-8.3.0/compose/AppStream/$(uname -p)/os/
+    rpm --import http://download.eng.brq.redhat.com/rhel-8/rel-eng/RHEL-8/latest-RHEL-8.3.0/compose/AppStream/$(uname -p)/os/RPM-GPG-KEY-redhat-release
+    dnf -y install java-headless
+    cd /etc/yum.repos.d/
+    rpm -i http://hdn.corp.redhat.com/rhel7-csb-stage/RPMS/noarch/redhat-internal-cert-install-0.1-23.el7.csb.noarch.rpm
+    curl -L -O https://download.devel.redhat.com/rel-eng/RCMTOOLS/rcm-tools-rhel-8-baseos.repo
+    dnf -y -y install brew-tools
```

Comment:
Please, don't use these repos on a public git repository. This is an upstream repository, use Copr instead. Thanks!

Reply:
If you need something enabled in CI, let me know.

Reply:
@ffmancera , I was unable to host kernel module in copr for EPEL-8 successfully, even if I can buld the kernel module rpm locally within Fedora ecosystem.   But I created the network connection via nmcli, and verified that `802-11-wireless-security.key-mgmt: sae` was appeared on the connection profile.